packages/core/contracts/oracle/interfaces/OptimisticOracleInterface.sol

```diff
@@ -37,7 +37,7 @@ abstract contract OptimisticOracleInterface {
 
     // This value must be <= the Voting contract's `ancillaryBytesLimit` value otherwise it is possible
     // that a price can be requested to this contract successfully, but cannot be disputed because the DVM refuses
-    // to accept a price request made with ancillary data length of a certain size.
+    // to accept a price request made with ancillary data length over a certain size.
```

Comment:
- In the `OptimisticOracleInterface` there is a comment about how the DVM could refuse to accept a dispute ["...with ancillary data length of a certain size."](https://github.com/UMAprotocol/protocol/blob/43d569f66af0323813716927ef2dcc8c7f60dd33/packages/core/contracts/oracle/interfaces/OptimisticOracleInterface.sol#L40). Something along the lines of: "...with ancillary data length *over* a certain size" may be more closely aligned with the implementation.